src/__tests__/EventHolder-test.js

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright 2013 Facebook, Inc.
+ * Copyright 2004-present Facebook. All Rights Reserved.
```

Comment:
(Yes, I know you got that bit from internal, but that's the fun of syncing - we can update internal with the right thing when we're all done)